linfa-trees/src/decision_trees/algorithm.rs

```diff
@@ -45,6 +49,24 @@ struct TreeNode {
     prediction: u64,
 }
 
+impl Hash for TreeNode {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        let mut stuff: Vec<u64> = vec![];
```

Comment:
nit: Is there a better name than `stuff` here? I haven't done an `impl Hash` before don't know the convention.